src/audio/lib/data.c

```diff
@@ -1,6 +1,6 @@
 #include "global.h"
 
-s16 gLowPassFilterData[16 * 8] = {
+s16 gLowPassFilterData[16 * 8] ALIGNED(16) = {
```

Comment:
What's the motivation for aligning all this audio data?

Reply:
The audio data gets DMA'd by the audio microcode, so if any of those variables somehow got misaligned then it would read garbage due to the DMAs constraints
Source: https://discord.com/channels/688807550715560050/688851451014938642/1173319172277682216 (tharo)